access-common/pom.xml

```diff
@@ -123,5 +130,11 @@
             <groupId>edu.unc.lib.cdr</groupId>
             <artifactId>fcrepo-clients</artifactId>
         </dependency>
-    </dependencies>
+      <dependency>
```

Comment:
You should be able to remove this dependency after making some of the changes suggested down in the loris service. I'd rather not get OSGI dependencies in here